src/Providers/ViewServiceProvider.php

```diff
@@ -172,6 +173,31 @@ private function registerRuntimeAntlers()
         });
     }
 
+    public function registerBladeDirectives()
+    {
+        Blade::directive('tags', function ($expression) {
```

Comment:
I think we make this testable by having a class that accepts the `$expression` and returns an array of the variables we want exposed back to the template.

Something like...

```php
Blade::directive('tags', function ($expression) {
    return "<?php extract(BladeTagsDirective::handle($expression)); ?>";
});
```

By using the `extract` method, any items in the returned array will be turned into variables. See https://www.php.net/manual/en/function.extract.php

Reply:
We'll write a test. No worries. Thanks for refactoring to the class though.